src/InkList.ts

```diff
@@ -379,6 +379,39 @@ export class InkList extends Map<SerializedInkListItem, number> {
 		else
 			return new InkList();
 	}
+	public ListWithSubRange(minBound: any, maxBound: any)
+	{
+		if (this.Count == 0) return new InkList();
+
+		let ordered = this.orderedItems;
+
+		let minValue = 0;
+		let maxValue = Number.MAX_SAFE_INTEGER;
+
+		if (Number.isInteger(minBound)) {
+			minValue = minBound;
```

Comment:
We're missing the `int` casts here and for `maxBound`.

Reply:
Happy to append `as Number`, but wouldn't it be redundant? We've already checked that `minBound: any` is an integer. In C# the cast is required, to coerce `object` to `int` but in TypeScript, `any` can always be implicitly coerced.

Reply:
Ah yes, you're correct. All good then!